### PR TITLE
test for search,edit and delete of apis(ONEPLAT-1724)

### PR DESCRIPTION
**Pull request by mudit94**
# Closes/Fixes/Resolves

### 

# Explain the feature/fix

## Does this PR introduce a breaking change

No

## Screenshots

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did tests pass?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: e2e/integration/apicatalogtests.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ Cypress.on( 'uncaught:exception', ( err, runnable ) => {
     // failing the test
     return false;
 } )
+let words;
```
Comment (riginoommen) [inline e2e/integration/apicatalogtests.js:7]:
```suggestion
```

File: e2e/integration/apicatalogtests.js [hunk 2]
```diff
@@ -20,7 +21,9 @@ context( 'API Catalog Tests', () => {
     it( 'Fill the add API form', () => {
         cy.faker = require( 'faker' );
         const words = cy.faker.lorem.words();
-        cy.get( 'input[name="name"]' ).type( words );
+        //this.words=words
```
Comment (riginoommen) [inline e2e/integration/apicatalogtests.js:24]:
```suggestion
```

File: e2e/integration/apicatalogtests.js [hunk 3]
```diff
@@ -20,7 +21,9 @@ context( 'API Catalog Tests', () => {
...
+        cy.get( 'input[name="name"]' ).type( 'e2e test automation' );
+        //cy.get( 'input[name="name"]' ).invoke('text').as('wordID')
```
Comment (riginoommen) [inline e2e/integration/apicatalogtests.js:26]:
```suggestion
```

File: e2e/integration/searchtests.js [hunk 4]
```diff
@@ -72,22 +73,22 @@ context( 'Test search', () => {
...
-        cy.get( 'input[name="query"]' ).click( { force: true } ).clear( { force: true } ).type( 'qwertyuiopasdgthchgbtnjkmnvxs' ,{force:true});
-        cy.get( '.op-search__btn' ).click();
-        cy.get( '#username', { timeout: 5000 } ).type( Cypress.env( 'USERNAME' ) );
-        cy.get( '#password' ).type( Cypress.env( 'PASSWORD' ) );
-        cy.get( '#submit' ).click();
+        cy.wait( 1500 );
+        cy.get( '.opc-nav-search__btn' ).click( { force: true } );
+        cy.wait( 500 );
+        cy.get( 'input[name="query"]' ).click( { force: true } ).clear( { force: true } ).type( 'qwertyuiopasdgthchgbtnjkmnvxs', { force: true } );
+        //cy.get( 'op-nav' ).find( 'input[name="query"]' ).click( { force: true } ).clear( { force: true } ).type( 'qwertyuiopasdgthchgbtnjkmnvxs', { force: true } );
```
Comment (riginoommen) [inline e2e/integration/searchtests.js:88]:
```suggestion
```

**Review by riginoommen — CHANGES_REQUESTED** [review]

File: e2e/integration/apicatalogtests.js (hunk 1 above)
Comment (mudit94) [reply e2e/integration/apicatalogtests.js]: resolved @riginoommen  https://github.com/1-Platform/one-platform/pull/1334/commits/579d9b4f9295f5fa94d2610890cf2007f2ebc0c4

File: e2e/integration/searchtests.js (hunk 4 above)
Comment (mudit94) [reply e2e/integration/searchtests.js:88]: https://github.com/1-Platform/one-platform/pull/1334/commits/579d9b4f9295f5fa94d2610890cf2007f2ebc0c4 resolved @riginoommen

File: e2e/integration/apicatalogtests.js (hunk 3 above)
Comment (mudit94) [reply e2e/integration/apicatalogtests.js]: https://github.com/1-Platform/one-platform/pull/1334/commits/579d9b4f9295f5fa94d2610890cf2007f2ebc0c4 resolved @riginoommen

File: e2e/integration/apicatalogtests.js (hunk 2 above)
Comment (mudit94) [reply e2e/integration/apicatalogtests.js]: resolved @riginoommen https://github.com/1-Platform/one-platform/pull/1334/commits/579d9b4f9295f5fa94d2610890cf2007f2ebc0c4

**Review by hybridx — APPROVED** [review]

**Review by hybridx — APPROVED** [review]

File: e2e/integration/searchtests.js (hunk 4 above)
Comment (mudit94) [reply e2e/integration/searchtests.js]: https://github.com/1-Platform/one-platform/pull/1334/commits/be6e9afef069a3aed36f8a7926bc6a2a1f90fb27 resolved @riginoommen

**Review by riginoommen — APPROVED** [review]

**Comment by riginoommen** [comment]
SQUASH the commits

**Pull request merged**